Chat/assignment-chat/client-chat/src/main/java/ru/assignment/net/ChatClient.java

```diff
@@ -0,0 +1,67 @@
+package ru.assignment.net;
+
+
+
+import java.io.IOException;
+import java.net.Socket;
+
+
+public class ChatClient {
+    private ClientConfiguration clientConfiguration;
+    private Sender sender;
+    private Receiver receiver;
+    private Socket clientSocket;
+
+    public static void main(String[] ar) {
+        ClientConfiguration clientConfiguration = new ClientConfiguration(8185, "localhost");
+        ChatClient chatClient = new ChatClient(clientConfiguration);
+        chatClient.startClient();
+    }
+
+    public ChatClient(ClientConfiguration clientConfiguration) {
+        this.clientConfiguration = clientConfiguration;
+    }
+
+    public void startClient() {
+        try {
+            initConfiguration();
+        } catch (IOException e) {
+            e.printStackTrace();
+            return;
+        }
+        System.out.println("beforestartSession");
+        startSession();
+        System.out.println("beforefinishSession");
+        finishSession();
+
+    }
+
+    public void initConfiguration() throws IOException {
+        int serverPort = clientConfiguration.getServerPort();
+        String serverHost = clientConfiguration.getServerHost();
+        clientSocket = new Socket(serverHost, serverPort);
+        sender = new Sender(clientSocket);
+        receiver = new Receiver(clientSocket, sender);
+    }
+
+    public void startSession() {
+        System.out.println("startSession");
```

Comment:
It will be more cool if you do something like:
    public void startSession() {
        receiver.startAsync();
        sender.startSender();
    }